collectors/scm/github-graphql/src/main/java/com/capitalone/dashboard/collector/DefaultGitHubClient.java

```diff
@@ -223,8 +230,23 @@ public void fireGraphQL(GitHubRepo repo, boolean firstRun, Map<Long, String> exi
         connectCommitToPulls();
     }
 
+    public RedirectedStatus checkForRedirectedRepo(GitHubRepo repo) throws MalformedURLException, HygieiaException {
+        GitHubParsed gitHubParsed = new GitHubParsed(repo.getRepoUrl());
+        String query = gitHubParsed.getBaseApiUrl() + "repos/" + gitHubParsed.getOrgName() + "/" + gitHubParsed.getRepoName();
+
+        ResponseEntity<String> response = makeRestCallGet(query);
+
+        JSONObject queryJSONBody = (JSONObject) parseAsObject(response);
+        String repoUrl = (String) queryJSONBody.get("html_url");
+        if (!repoUrl.equals(repo.getRepoUrl())) {
+            LOG.debug("original url: " + repo.getRepoUrl() + " is redirected to new url: " + repoUrl);
```

Comment:
To be splunk/ELK friendly, I would use LOG.info instead of LOG.debug, and use the format "URL redirection: original=<url>, redirected=<url>"